readme.md

```diff
@@ -90,23 +92,23 @@ Once installation is working, try creating and requesting resources. Note that t
 
 * **Question:** Describe what the above code is adding to your schema.
 
-* **Your Answer:**
+* **Your Answer:** Adds the time values for when the reocrd was created and updated.
 
 ---
 
 - [ ] Import the model into your `routes/series.js` file. Use either basic promises or `async/await` to update the `GET /` method to retrieve from the database instead of the given array. [This page](https://mongoosejs.com/docs/documents.html) or [this page](https://mongoosejs.com/docs/api.html#Model) may be useful.
 
 * **Question:** What method do you use to return all documents from a collection?
 
-* **Your Answer:**
+* **Your Answer:** GET http://localhost:5000/api/series
```

Comment:
This is the _route_ for returning all documents. With Mongoose, you'll use `.find()` to return all documents in a collection.